unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/DependencyMapLoadHelper.cs

```diff
@@ -62,31 +62,28 @@ public static IEnumerator GetDepMap(string baseUrl, string hash)
             yield break;
         }
 
-        using (UnityWebRequest depmapRequest = UnityWebRequest.Get(url))
-        {
-            downloadingDepmap.Add(hash);
-
-            yield return depmapRequest.SendWebRequest();
-
-            if (!depmapRequest.WebRequestSucceded())
+        downloadingDepmap.Add(hash);
+        WebRequestController.i.Get(
```

Comment:
This request was yielded for a reason. If we don't yield, the method will return before the request is finished and this may be unexpected from the AssetPromise.

Reply:
My fault, I didn't notice when I applied the change of method. Thanks! Done in https://github.com/decentraland/explorer/pull/2256/commits/d24a7ebe1a73c92facab4dc8258aeba4f3a7368c